gameboy.h

```diff
@@ -817,10 +823,14 @@ void gb_reset(struct gb_s *gb)
     gb->gb_reg.SC = 0x7E;
     gb->gb_reg.STAT = 0;
     gb->gb_reg.LY = 0;
-
+    /* Avoid using __gb_write when instruction tests, since __gb_write() is called to
+     * write data to shared memory with gbit functions.
```

Comment:
The intention of macro `DEBUG` was not for GBIT. Please check the source code again for understanding the general ideas. Instead, you should specify yet another macro associated with GBIT.